### PR TITLE
define get_highest_status() under BadgeStatus

### PR DESCRIPTION
**Pull request by ylil93**
- refactor code to use this method

### CODE REVIEW

File: badge_server/main.py [hunk 1]
```diff
@@ -76,6 +77,23 @@ class BadgeStatus(enum.Enum):
...
     SUCCESS = 'SUCCESS'
 
+    @classmethod
+    def get_highest_status(cls, statuses: Iterable[enum.Enum]):
+        """Gets the highest BadgeStatus.
+
+        This method works because `cls.__members__` is an ordered dict.
+
+        Args:
+            statuses: A list of BadgeStatuses.
```
Comment (brianquinlan) [inline badge_server/main.py]:
```suggestion
            statuses: An interable of BadgeStatuses. The highest priority package in this iterable will be returned.
```

File: badge_server/main.py [hunk 2]
```diff
@@ -76,6 +77,23 @@ class BadgeStatus(enum.Enum):
...
+
+        Returns:
+            A BadgeStatus. An error is thrown if no BadgeStatus exists.
```
Comment (brianquinlan) [inline badge_server/main.py]: http://google.github.io/styleguide/pyguide.html#244-decision

```suggestion
            The BadgeStatus found in `statuses` that has the highest priority. For example, INTERNAL_ERROR would be returned if `statuses` contained INTERNAL_ERROR and OUTDATED_DEPENDENCY.
            
  Raises:
     ValueError: if `statuses` does not contain a valid BadgeStatus.
```

File: badge_server/main.py (hunk 1 above)
Comment (ylil93) [reply badge_server/main.py]: done

File: badge_server/main.py (hunk 2 above)
Comment (ylil93) [reply badge_server/main.py]: done

File: badge_server/main.py [hunk 3]
```diff
@@ -76,6 +77,29 @@ class BadgeStatus(enum.Enum):
...
+        Args:
+            statuses: A list of BadgeStatuses.
+
+        Returns:
+            The BadgeStatus found in `statuses` that has the highest priority.
+            For example, INTERNAL_ERROR would be returned if `statuses`
+            contained INTERNAL_ERROR and OUTDATED_DEPENDENCY.
+
+        Raises:
+            ValueError if no BadgeStatus exists. For example, if the length of
```
Comment (brianquinlan) [inline badge_server/main.py]:
```suggestion
            ValueError: if no BadgeStatus exists. For example, if the length of
```

Comment (ylil93) [reply badge_server/main.py]: done

**Pull request merged**